README.md

```diff
@@ -6,8 +6,73 @@ Samba metrics exporter converts 'smbstatus' output into
 
 ## Build
 
-```bash
+```sh
```

Comment:
I think `console`is preferred over `bash`and `sh`for shell output. the others are for scripts.

Reply:
Fair enough -- fixed.